src/GraphQL/Types/Composite/InputObjectGraphType.cs

```diff
@@ -1,10 +1,16 @@
+using System.Collections.Generic;
+
 namespace GraphQL.Types
 {
     /// <summary>
     /// Represents an input object graph type.
     /// </summary>
     public interface IInputObjectGraphType : IComplexGraphType
     {
+        /// <summary>
+        /// Converts a supplied dictionary of keys and values to an object.
+        /// </summary>
+        object ParseDictionary(IDictionary<string, object> value);
```

Comment:
ConvertDictionary ?

Reply:
I chose `ParseDictionary` because it "matches" `ParseLiteral` and `ParseValue`.  It's only called while parsing arguments and variables.  Do you think `ConvertDictionary` is better?

Reply:
OK, ParseDictionary